public/favicon/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
```

Comment:
I also added a new SWC favicon based on the logo 😄

Reply:
👍 Thanks